react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+const topics = [
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Bulbasaur' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Charmander' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Squirtle' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Pikachu' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Jigglypuff' }
+];
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: 'false' };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    if (event.target.className === 'closed') {
+      for (let x = 0; x < event.target.parentElement.parentElement.children.length; x++) {
+        event.target.parentElement.parentElement.children[x].children[0].className = 'closed';
+        event.target.parentElement.parentElement.children[x].children[1].className = 'hidden';
+        event.target.className = 'open';
+        event.target.nextElementSibling.className = 'visible';
+      }
+    } else {
+      event.target.className = 'closed';
+      event.target.nextElementSibling.className = 'hidden';
+    }
+  }
```

Comment:
In React we want to avoid direct DOM manipulation by adding and removing classes in this way. Here your `handleClick` just needs to get the `index` of the topic that was clicked on, and set that to state.
```JAVASCRIPT
handleClick(index) {
  if (this.state.activeIndex === index) {
    this.setState({ activeIndex: null });
  } else {
    this.setState({ activeIndex: index });
  }
}
```